packages/next/lib/load-custom-routes.ts

```diff
@@ -510,6 +510,13 @@ export default async function loadCustomRoutes(
     }
   }
 
+  // multiple slashes redirect
+  redirects.unshift({
+    source: '/:before*/{(/+)}:after(|.*/[^/.]+|[^/.]+)',
+    destination: '/:before*/:after',
+    permanent: true,
```

Comment:
```suggestion
    permanent: true,
    locale: config.i18n ? false : undefined,
```

We need to specify `locale: false` like above so it's not included in the redirect when it wasn't in the source since it's always included when not `false`